Getting-Started.md

```diff
@@ -1,3 +1,191 @@
+- [SortingHat](https://github.com/chaoss/grimoirelab-sortinghat)
+- [Sigils](https://github.com/chaoss/grimoirelab-sigils)
+- [Kidash](https://github.com/chaoss/grimoirelab-kidash)
+- [Toolkit](https://github.com/chaoss/grimoirelab-toolkit)
+- [Cereslib](https://github.com/chaoss/grimoirelab-cereslib)
+- [Manuscripts](https://github.com/chaoss/grimoirelab-manuscripts)
+
+
+Each local repo should have two `remotes`: `origin` points to the forked repo, while `upstream` points to the original CHAOSS repo. An example
+is provided below.
+```
+~/sources/perceval$ git remote -v
+origin	https://github.com/valeriocos/perceval (fetch)
+origin	https://github.com/valeriocos/perceval (push)
+upstream	https://github.com/chaoss/grimoirelab-perceval (fetch)
+upstream	https://github.com/chaoss/grimoirelab-perceval (push)
+```
+
+In order to add a remote to a Git repository, you can use the following command:
+```
+~/sources/perceval$ git remote add upstream https://github.com/chaoss/grimoirelab-perceval
+```
+
+Then, you can download the [PyCharm community edition](https://www.jetbrains.com/pycharm/download/#section=linux), and create a project in the 
+grimoirelab-sirmordred directory. PyCharm will automatically create a virtual env, where you should install the dependencies listed in each 
+requirements.txt, excluding the ones concerning the grimoirelab components.
+
+To install the dependencies, you can click on `File` -> `Settings` -> `Project` -> `Project Interpreter`, and then the `+` located on the top right corner (see figure below).
+
+![captura_22](https://user-images.githubusercontent.com/6515067/63195511-12299d80-c073-11e9-9774-5f274891720a.png)
+
+Later, you can add the dependencies to the grimoirelab components via `File` -> `Settings` -> `Project` -> `Project Structure`. 
+The final results should be something similar to the image below.
+
+![captura_23](https://user-images.githubusercontent.com/6515067/63195579-4b620d80-c073-11e9-888b-3cdb67c04523.png)
+
+Finally, you can use the docker-compose shown at Section [source-code-and-docker](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/README.md#source-code-and-docker), define a [setup.cfg](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/utils/setup.cfg) and [projects.json](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/utils/projects.json), and
+run the following commands, which will collect and enrich the data coming from the git and cocom sections and upload the corresponding panels to Kibiter:
+```
+micro.py --raw --enrich --cfg ./setup.cfg --backends git cocom
+micro.py --panels --cfg ./setup.cfg
+```
+
+Optionally, you can create a configuration in PyCharm to speed up the executions (`Run` -> `Edit configuration` -> `+`). The final results should be something similar to the image below.
+
+![captura_24](https://user-images.githubusercontent.com/6515067/63195767-ccb9a000-c073-11e9-805a-e828a3ce1dc9.png)
+ 
```

Comment:
How about a picture of the dashboard result when micro.py is executed, in the end?

Reply:
well have to ask @valeriocos for that, I don't think the addition is needed though. Sorry :). @valeriocos your decision is final.

Reply:
@vchrombie what do you think if this image is added here: https://github.com/chaoss/grimoirelab#grimoirelab ?

Reply:
Hi @valeriocos @abhiandthetruth 

> How about a picture of the dashboard result when micro.py is executed, in the end?

I was thinking of just having the result of running micro-mordred (maybe with just Git or Github backend) here, as we had the pictures of all the installation steps. Anyway, this can be omitted. 

> @vchrombie what do you think if this image is added here: https://github.com/chaoss/grimoirelab#grimoirelab ?

Even that would be a good place to add. How about the screenshot of [CHAOSS Dashboard](https://chaoss.biterg.io/) over there, as this dashboard has many data sources?

Reply:
> I was thinking of just having the result of running micro-mordred (maybe with just Git or Github backend) here, as we had the pictures of all the installation steps. Anyway, this can be omitted.

Please check the comment at https://github.com/chaoss/grimoirelab-sirmordred/pull/428#discussion_r392673572. Maybe we can add the pictures in the section `execution`, or we can have them in a dedicated section called `example/showcase`. WDYT?

> Even that would be a good place to add. How about the screenshot of CHAOSS Dashboard over there, as this dashboard has many data sources?

+1

Reply:
For me `example/showcase` sounds better. WDYT @vchrombie?

Reply:
Sorry, I will not be able to do it here, since this pr is focused on restructuring readme, I leave the addition of `example/showcase` to @vchrombie. All the best :)

Reply:
Sure, I will do that.
Sorry for the late reply.